activiti-cloud-services-query/activiti-cloud-services-query-rest/src/main/java/org/activiti/cloud/services/query/rest/TaskAdminController.java

```diff
@@ -87,6 +89,16 @@ public String handleAppException(IllegalStateException ex) {
     public PagedResources<TaskResource> allTasks(@QuerydslPredicate(root = TaskEntity.class) Predicate predicate,
                                                  Pageable pageable) {
 
+        //when request is made for the root tasks only replace (= null) with (is null)
+        String s = predicate !=null ? predicate.toString() : null;
+        if (s != null) {
+            if (s.contains("parentTaskId = null")) {
```

Comment:
@CTI777 I would hide the `null` from the user.. for example. I would do something like 
``
v1/tasks/search?rootTasksOnly  (or rootTasksOnly=true)
```
So at least we hide the implementation details and the semantic meaning for the filter makes more sense. 

Does it makes sense? I am happy to merge if you do that change..

Reply:
Also look at the suggestion from codacy regarding the null and contains check: 
https://app.codacy.com/app/Activiti/activiti-cloud-query-service/pullRequest?prid=2804221

Reply:
@salaboy 
Can we make it in this way: v1/tasks/rootTasksOnly?value1={value1}&value2={value2}.
In this way we have standard way for predicate and a special condition for rootTasks?
I have some problems with v1/tasks?rootTasksOnly, because it affects get all tasks...

Reply:
@CTI777 yes.. I think this is ok for now.. let's merge it as it is